gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/drilling/Drill.java

```diff
@@ -222,5 +223,16 @@ public DrillUp buildDrillUp() {
       return new DrillUp(
         label, propertyKey, function, newPropertyKey, drillVertex);
     }
+
+    /**
+     * Creates a drill down operation.
+     *
+     * @return drill down operation
+     */
+    public DrillDown buildDrillDown() {
+      Objects.requireNonNull(propertyKey);
```

Comment:
My intention was to ensure that there is a property key defined. And if this checkup is made in the setter but the setter is never used this will cause a npe.